fuse_viz/src/relative_pose_2d_stamped_constraint_visual.cpp

```diff
@@ -343,7 +343,13 @@ Ogre::ColourValue RelativePose2DStampedConstraintVisual::computeLossErrorLineCol
   // Get the error line color as HSB:
   Ogre::ColourValue error_line_color(color.r, color.g, color.b);
   Ogre::Real hue, saturation, brightness;
+#if (OGRE_VERSION < ((1 << 16) | (11 << 8) | 0))
```

Comment:
Please include the `OgrePrerequisites.h` header to support the `OGRE_VERSION` macro.

Reply:
Done in https://github.com/locusrobotics/fuse/pull/270/commits/5b4dd949514cef2a89dbcd02003094ba186b5262